tests/snuba/api/endpoints/test_organization_events_stats_span_metrics.py

```diff
@@ -192,6 +192,38 @@ def test_top_events(self):
         assert "bar" in response.data
         assert response.data["Other"]["meta"]["dataset"] == "spansMetrics"
 
+    def test_resource_size(self):
+        for i in range(10):
```

Comment:
nit: do we need 10 for this test case? We don't need to test the average function in snuba, so even 1 event here is enough

Reply:
Yeah that's a good point, if avg function is already tested outside of this we're good. I'll refactor so its only 1 event.